covid_tracking/app.py

```diff
@@ -315,11 +334,18 @@ def update_states_map(metric, pc, map_mode='scatter'):
         raise ValueError(f'Map mode must be one of "choropleth" or "scatter" (not "{map_mode}")')
     if metric is None:
         metric = DEFAULT_METRIC
+
     if pc:
         metric = metric + '_pc'
 
-    df = api.states_current
-    df = df[df[metric] > 0]
+    # Getting today's new cases
+    if 'newCases' in metric:
+        df = api.states_daily
+        df = df[df['date'] == int(datetime.datetime.today().strftime('%Y%m%d'))]
```

Comment:
Does this work if it's currently March 23rd and the most recent data from the API only goes up to the 22nd?  It looks to me like this would return nothing for a lot of the day.